src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -159,8 +159,13 @@ setTimeout(async function () {
       });
     }
 
+    let body = JSON.stringify({
+      "id": deltas[0].sequenceId,
```

Comment:
Do we know for certain that `deltas` always contains at least one entry?

Reply:
The client shouldn't be sending updates with empty deltas.